.gitattributes

```diff
@@ -0,0 +1,18 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Explicitly declare text files you want to always be normalized and converted
+# to native line endings on checkout.
+*.md text
+*.txt text
+*.yaml text
```

Comment:
Do we always use .yaml or do we need to include .yml here?

Reply:
It's always `.yaml` in this repo